docroot/profiles/custom/cgov_site/modules/custom/cgov_list/cgov_list.module

```diff
@@ -106,6 +127,11 @@ function cgov_list_allowed_values_function(
 function cgov_list_paragraph_view(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
   $bundle = $entity->bundle();
   if ($bundle === 'cgov_list') {
+    $node = \Drupal::routeMatch()->getParameter('node');
```

Comment:
So first, this needs a comment to explain the why here. Secondly, what is this doing??

Reply:
@dev-rana-publicis This is explaining the what, not the why. I was about to type what the comment should be, and then I figured the why I came up with was not right further nailing home that this needs the why not the what.